internal_review/economics/citywide/ACS_PUMS_economics_0812.csv

```diff
@@ -0,0 +1,2 @@
ct_moe,mwedhlt_bnh_count,mwedhlt_bnh_count_moe,mwedhlt_bnh_count_cv,mwedhlt_bnh_pct,mwedhlt_bnh_pct_moe,mwarten_bnh_count,mwarten_bnh_count_moe,mwarten_bnh_count_cv,mwarten_bnh_pct,mwarten_bnh_pct_moe,mwoth_bnh_count,mwoth_bnh_count_moe,mwoth_bnh_count_cv,mwoth_bnh_pct,mwoth_bnh_pct_moe,mwpbadm_bnh_count,mwpbadm_bnh_count_moe,mwpbadm_bnh_count_cv,mwpbadm_bnh_pct,mwpbadm_bnh_pct_moe,cvem2_hsp_count,cvem2_hsp_count_moe,cvem2_hsp_count_cv,cvem2_hsp_pct,cvem2_hsp_pct_moe,mwagff_hsp_count,mwagff_hsp_count_moe,mwagff_hsp_count_cv,mwagff_hsp_pct,mwagff_hsp_pct_moe,mwcnstn_hsp_count,mwcnstn_hsp_count_moe,mwcnstn_hsp_count_cv,mwcnstn_hsp_pct,mwcnstn_hsp_pct_moe,mwmnfct_hsp_count,mwmnfct_hsp_count_moe,mwmnfct_hsp_count_cv,mwmnfct_hsp_pct,mwmnfct_hsp_pct_moe,mwwhlsl_hsp_count,mwwhlsl_hsp_count_moe,mwwhlsl_hsp_count_cv,mwwhlsl_hsp_pct,mwwhlsl_hsp_pct_moe,mwrtl_hsp_count,mwrtl_hsp_count_moe,mwrtl_hsp_count_cv,mwrtl_hsp_pct,mwrtl_hsp_pct_moe,mwtrwhu_hsp_count,mwtrwhu_hsp_count_moe,mwtrwhu_hsp_count_cv,mwtrwhu_hsp_pct,mwtrwhu_hsp_pct_moe,mwinfo_hsp_count,mwinfo_hsp_count_moe,mwinfo_hsp_count_cv,mwinfo_hsp_pct,mwinfo_hsp_pct_moe,mwfire_hsp_count,mwfire_hsp_count_moe,mwfire_hsp_count_cv,mwfire_hsp_pct,mwfire_hsp_pct_moe,mwpfmg_hsp_count,mwpfmg_hsp_count_moe,mwpfmg_hsp_count_cv,mwpfmg_hsp_pct,mwpfmg_hsp_pct_moe,mwedhlt_hsp_count,mwedhlt_hsp_count_moe,mwedhlt_hsp_count_cv,mwedhlt_hsp_pct,mwedhlt_hsp_pct_moe,mwarten_hsp_count,mwarten_hsp_count_moe,mwarten_hsp_count_cv,mwarten_hsp_pct,mwarten_hsp_pct_moe,mwoth_hsp_count,mwoth_hsp_count_moe,mwoth_hsp_count_cv,mwoth_hsp_pct,mwoth_hsp_pct_moe,mwpbadm_hsp_count,mwpbadm_hsp_count_moe,mwpbadm_hsp_count_cv,mwpbadm_hsp_pct,mwpbadm_hsp_pct_moe,cvem2_anh_count,cvem2_anh_count_moe,cvem2_anh_count_cv,cvem2_anh_pct,cvem2_anh_pct_moe,mwagff_anh_count,mwagff_anh_count_moe,mwagff_anh_count_cv,mwagff_anh_pct,mwagff_anh_pct_moe,mwcnstn_anh_count,mwcnstn_anh_count_moe,mwcnstn_anh_count_cv,mwcnstn_anh_pct,mwcnstn_anh_pct_moe,mwmnfct_anh_count,mwmnfct_anh_count_moe,mwmnfct_anh_count_cv,mwmnfct_anh_pct,mwmnfct_anh_pct_moe,mwwhlsl_anh_count,mwwhlsl_anh_count_moe,mwwhlsl_anh_count_cv,mwwhlsl_anh_pct,mwwhlsl_anh_pct_moe,mwrtl_anh_count,mwrtl_anh_count_moe,mwrtl_anh_count_cv,mwrtl_anh_pct,mwrtl_anh_pct_moe,mwtrwhu_anh_count,mwtrwhu_anh_count_moe,mwtrwhu_anh_count_cv,mwtrwhu_anh_pct,mwtrwhu_anh_pct_moe,mwinfo_anh_count,mwinfo_anh_count_moe,mwinfo_anh_count_cv,mwinfo_anh_pct,mwinfo_anh_pct_moe,mwfire_anh_count,mwfire_anh_count_moe,mwfire_anh_count_cv,mwfire_anh_pct,mwfire_anh_pct_moe,mwpfmg_anh_count,mwpfmg_anh_count_moe,mwpfmg_anh_count_cv,mwpfmg_anh_pct,mwpfmg_anh_pct_moe,mwedhlt_anh_count,mwedhlt_anh_count_moe,mwedhlt_anh_count_cv,mwedhlt_anh_pct,mwedhlt_anh_pct_moe,mwarten_anh_count,mwarten_anh_count_moe,mwarten_anh_count_cv,mwarten_anh_pct,mwarten_anh_pct_moe,mwoth_anh_count,mwoth_anh_count_moe,mwoth_anh_count_cv,mwoth_anh_pct,mwoth_anh_pct_moe,mwpbadm_anh_count,mwpbadm_anh_count_moe,mwpbadm_anh_count_cv,mwpbadm_anh_pct,mwpbadm_anh_pct_moe
```

Comment:
@SashaWeinstein  the code has the changes with the `edu_` so I am assuming you didn't write new internal review outputs correct?